include/zephyr/drivers/misc/renesas_rx_dtc/renesas_rx_dtc.h

```diff
@@ -0,0 +1,238 @@
+ * @param p_info transfer info.
+ *
+ * @retval 0 On success.
+ * @retval -EINVAL if activation source is invalid.
+ */
+int dtc_renesas_rx_configuration(const struct device *dev, uint8_t activation_irq,
+				 transfer_info_t *p_info);
+
+/**
+ * @brief Enable transfer in ICU on this activation source.
+ *
+ * @param activation_irq activation source.
+ *
+ * @retval 0 On success.
+ * @retval -EINVAL if activation source is invalid.
+ */
+int dtc_renesas_rx_enable_transfer(uint8_t activation_irq);
+
+/**
+ * @brief Disable transfer in ICU on this activation source.
+ *
+ * @param activation_irq activation source.
+ *
+ * @retval 0 On success.
+ * @retval -EINVAL if activation source is invalid.
+ */
+int dtc_renesas_rx_disable_transfer(uint8_t activation_irq);
+
+/**
+ * @brief Start transfers on this activation source.
+ *
+ * @param dev DTC device instance.
+ * @param activation_irq activation source.
+ *
+ * @retval 0 On success.
+ * @retval -EINVAL if activation source is invalid.
+ * @retval -EACCES if this activation source in dtc vector table is not configured.
+ */
+int dtc_renesas_rx_start_transfer(const struct device *dev, uint8_t activation_irq);
+
+/**
+ * @brief Stop transfers on this activation source.
+ *
+ * @param dev DTC device instance.
+ * @param activation_irq activation source.
+ *
+ * @retval 0 On success.
+ * @retval -EINVAL if activation source is invalid.
+ * @retval -EACCES if this activation source in dtc vector table is not configured.
+ */
+int dtc_renesas_rx_stop_transfer(const struct device *dev, uint8_t activation_irq);
+
+/**
+ * @brief Reset transfer on this activation source.
+ *
+ * @param dev             DTC device instance.
+ * @param activation_irq  Activation source IRQ number.
+ * @param p_src           Pointer to the source address for the transfer.
+ * @param p_dest          Pointer to the destination address for the transfer.
+ * @param num_transfers   Number of data units to transfer.
+ *
+ * @retval 0              On success.
+ * @retval -EINVAL        If activation source is invalid.
+ */
+int dtc_renesas_rx_reset_transfer(const struct device *dev, uint8_t activation_irq,
+				  void const *p_src, void *p_dest, uint16_t const num_transfers);
+
+/**
+ * @brief Get status transfer of DTC module and store it into status pointer.
+ *
+ * @param dev DTC device instance.
+ * @param status DTC status.
+ */
+void dtc_renesas_rx_get_transfer_status(const struct device *dev,
+					struct dtc_transfer_status *status);
+
+/**
+ * @brief Get information about this transfer and store it into p_properties.
+ *
+ * @param dev               DTC device instance.
+ * @param activation_irq    Activation source IRQ number.
+ * @param p_properties      Pointer to structure to receive driver-specific transfer properties.
+ *
+ * @retval 0        On success.
+ * @retval -EINVAL  If activation source is invalid.
+ */
+int dtc_renesas_rx_infoGet(const struct device *dev, uint8_t activation_irq,
```

Comment:
Use snake case for the api
```suggestion
int dtc_renesas_rx_info_get(const struct device *dev, uint8_t activation_irq,
```

Reply:
Updated